.env.default

```diff
@@ -0,0 +1,3 @@
+MAILGUN_API_KEY=
```

Comment:
Would be nice to put samples here so people can expect something. e.g. `MAILGUN_API_KEY=123567890abcdef` (pad it to the correct length)